src/main/java/net/mersid/mixin/GameRendererMixin.java

```diff
@@ -0,0 +1,40 @@
+package net.mersid.mixin;
+
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import net.mersid.events.CameraShakeEventListener;
+import net.mersid.events.CameraShakeEventListener.CameraShakeEvent;
+import net.minecraft.client.render.GameRenderer;
+import net.wurstclient.WurstClient;
+
+/**
+ * Runs every tick to determine if the player is hurt. Canceling will prevent the camera shake, but does not tell anything about WHEN
+ * the player gets hurt.
+ * @author Mersid
+ *
+ */
+@Mixin(GameRenderer.class)
+public class GameRendererMixin {
+	
+	/**
+	 * An event is fired here which, if canceled, will stop camera shake.
+	 * @param partialTicks
+	 * @param ci
+	 */
+	@Inject(at = @At("HEAD"),

```

Comment:
As an added note, this will not inject, I suspect it will CTD. Try this: 
`@Inject(at = @At("HEAD"), method = "bobViewWhenHurt(Lnet/minecraft/client/util/math/MatrixStack;F)V", cancellable = true)`